File: android_app/app/src/main/java/com/oligark/flashapp/model/Service.java
package com.oligark.flashapp.model;

public class Service {
    private int id;
    private String nombreServicio;

    public Service(int id, String nombreServicio) {
        this.id = id;
        this.setNombreServicio(nombreServicio);
    }

    public String getNombreServicio() {
        return nombreServicio;
    }

    public void setNombreServicio(String nombreServicio) {
        this.nombreServicio = nombreServicio;
    }
}

File: android_app/app/src/main/java/com/oligark/flashapp/model/Service.kt


package com.oligark.flashapp.model

import java.util.*

data class Service(
        var id: Long?=null,
        var date: Date?=null,
        var comments: String?=null,
        var status: ServiceStatus?=null,
        var serviceSchedule: ServiceSchedule?=null
) {
}